web/app/scripts/map-layers/recent-events/recent-events-layer-directive.js

```diff
@@ -74,25 +73,20 @@
             var recordsLayerOptions = angular.extend(defaultLayerOptions, {zIndex: 3});
             var occurredMin = new Date();
             occurredMin.setDate(occurredMin.getDate() - recencyCutoffDays);
-            // TODO: Boundary filtering somewhere in here.
-            // TODO: Since RecordType selection is going away, this may have to change
             RecordState.getSelected().then(function(selected) {
                 return TileUrlService.recTilesUrl(selected.uuid);
             // Construct Windshaft URL
             }).then(function(baseUrl) {
-                // TODO: Change and uncomment boundary filtering whenever Windshaft filtering is fixed.
-                return BoundaryState.getSelected().then(function(/*boundary*/) {
-                    return QueryBuilder.unfilteredDjangoQuery(0,
-                        {query: true,
-                         /* jshint camelcase: false */
-                         occurred_min: occurredMin.toISOString(),
-                         //polygon_id: boundary.uuid
-                         /* jshint camelcase: true */
-                        }
-                    ).then(function(result) {
-                            var queryParam = baseUrl.match(/\?/) ? '&sql=' : '?sql=';
-                            var sql = encodeURIComponent(result.query);
-                            return baseUrl + queryParam + sql;
+                return BoundaryState.getSelected().then(function(boundary) {
+                    return QueryBuilder.unfilteredDjangoQuery(0, {
+                        tilekey: true,
+                        /* jshint camelcase: false */
+                        occurred_min: occurredMin.toISOString(),
+                        polygon_id: boundary.uuid
+                        /* jshint camelcase: true */
+                    }).then(function(result) {
+                        var tilekeyParam = (baseUrl.match(/\?/) ? '&' : '?') + 'tilekey=';
+                        return baseUrl + tilekeyParam + result.tilekey;
```

Comment:
Yeah, they're just UUIDs (numbers, letters, hyphens), so it didn't seem necessary.